integrations/import_keys.go

```diff
@@ -0,0 +1,98 @@
+package integrations
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/olekukonko/tablewriter"
+	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/cli/utils"
+	"github.com/Scalingo/go-scalingo"
+)
+
+func ImportKeys(integration string) error {
+	var id string
+	var name string
+	var keys []scalingo.Key
+
+	c, err := config.ScalingoClient()
+	if err != nil {
+		return errgo.Notef(err, "fail to get Scalingo client")
+	}
+
+	if !utils.IsUUID(integration) {
+		i, err := integrationByName(c, integration)
+		if err != nil {
+			return errgo.Mask(err)
```

Comment:
`errgo.Mask` is banished. Still used here and there but please don't use it in new code. Use `errgo.Notef` instead